src/DateTime/tests/test_datetime.py

```diff
@@ -683,6 +683,22 @@ def test_security(self):
         self.assertEqual(dt.__roles__, None)
         self.assertEqual(dt.__allow_access_to_unprotected_subobjects__, 1)
 
+    @unittest.skipUnless(PY3K, 'format method is Python 3 only')
+    def test_format(self):
+        dt = DateTime(1968, 3, 10, 23, 45, 0, 'Europe/Vienna')
+        fmt = '%-d.%-m.%Y %H:%M'
+        result = dt.strftime(fmt)
+        unformatted_result = '1968/03/10 23:45:00 Europe/Vienna'
+        self.assertEqual(result, '{:%-d.%-m.%Y %H:%M}'.format(dt))
+        self.assertEqual(unformatted_result, '{:}'.format(dt))
+        self.assertEqual(unformatted_result, '{}'.format(dt))
+        if sys.version_info > (3, 6, 0):  # pragma: no cover
+            eval("self.assertEqual(result, f'{dt:{fmt}}')")
+            eval("self.assertEqual(unformatted_result ,f'{dt:}'.format(dt))")
+            eval("self.assertEqual(unformatted_result, f'{dt}'.format(dt))")
```

Comment:
What's the reason behind using an fstring together with `format`?
I'd expect that the fstring already does the formatting and the `format` call does nothing.